mmcv/utils/config.py

```diff
@@ -253,6 +253,32 @@ def fromfile(filename,
             import_modules_from_strings(**cfg_dict['custom_imports'])
         return Config(cfg_dict, cfg_text=cfg_text, filename=filename)
 
+    @staticmethod
+    def fromstring(cfg_str, suffix=None, is_pretty_text=True):
```

Comment:
The argument `is_pretty_text` is unnecessary. Setting `suffix='py'` or `is_pretty_text=True` are of the same cost for users.

Reply:
If remove `is_pretty_text`, when the user input `suffix='.json'` and cfg_str not from pretty_text, the program will go wrong.